NineChronicles.Headless.Executable/Program.cs

```diff
@@ -29,6 +29,7 @@ namespace NineChronicles.Headless.Executable
     [HasSubCommands(typeof(ValidationCommand), "validation")]
     [HasSubCommands(typeof(ChainCommand), "chain")]
     [HasSubCommands(typeof(KeyCommand), "key")]
+    [HasSubCommands(typeof(ConversionCommand), "conversion")]
```

Comment:
I guess `conversion` features relate to `key`, so it's more natural for me that integrate with `key`. (or merging them into `account`)

Reply:
How way do you think to merge them? Can you give me an example? In my thought, there is no relationship because it does only conversion without dependency with key store.

Reply:
> there is no relationship because it does only conversion without dependency with key store.

Technically yes, they don't have any relationship. but from the user-side, they want to convert private key to address (or reverse) when managing their Keystore IMO. considering this context, `key` and `conversion` features can be related.

> Can you give me an example?

I imagined like as following...

```bash
$ NineChronicles.Headless.Executable key convert --address <private-key> 
$ NineChronicles.Headless.Executable key convert --public-key ​<private-key>
```

Also, even though features belonging to `conversion` is not directly related to `key`, I think the name `conversion` is too general.

Reply:
Thanks to describe it. And I understood. Last one, I wondered one point, do you belive this command structure can be applied also to `Libplanet.Tools`?

Reply:
> do you belive this command structure can be applied also to Libplanet.Tools?

Yes 😅

Reply:
I applied it in 1e1bfe8 and 5679a22. Could you check it, please? 🙏

Reply:
Sadly, the logic was wrong, I will comment you after I fix.

Reply:
Could you review again, please? You can see 1e1bfe8, 5afdd48.